src/main/java/qna/service/QnaService.java

```diff
@@ -34,26 +34,44 @@ public QnaService(QuestionRepository questionRepository, AnswerRepository answer
 
     @Transactional(readOnly = true)
     public Question findQuestionById(Long id) {
-        return questionRepository.findByIdAndDeletedFalse(id)
-                .orElseThrow(NotFoundException::new);
+        return questionRepository.findByIdAndDeletedFalse(id).orElseThrow(NotFoundException::new);
     }
 
     @Transactional
     public void deleteQuestion(User loginUser, Long questionId) throws CannotDeleteException {
```

Comment:
```suggestion
    public void deleteQuestion(User loginUser, Long questionId) {
```

CannotDeleteException 클래스를 unchecked exception으로 바꿔보시기 바랍니다.